### DISCUSSION TITLE
Getting error "There was an error validating your contract, please try again" while verifying the smart contract

### DISCUSSION THREAD

**Discussion by L2nitant** [blockscout/blockscout#8976] [Q&A]
I started the blockscout earlier and after that I started docker for verification smart contract(while blockscout was already running)
my chain is running perfectly fine and blocks are shows in the explorer. But When I am trying to verify the smart contract, it shows me this error. 
`There was an error validating your contract, please try again`
My env variable are 
```
export DATABASE_URL=postgres url
export SECRET_KEY_BASE=secret key
export ETHEREUM_JSONRPC_VARIANT=geth
export ETHEREUM_JSONRPC_HTTP_URL=https://racetestnet.io/
export MIX_ENV=dev
export SHOW_TESTNET_LABEL=true
export TESTNET_LABEL_TEXT=Testnet
export NETWORK=TESTNET
export SUBNETWORK=RACE
export HIDE_BLOCK_MINER=true
export MICROSERVICE_SC_VERIFIER_ENABLED=true
export MICROSERVICE_SC_VERIFIER_URL=http://0.0.0.0:8050/
export CHAIN_ID=90001
export JSON_RPC=https://racetestnet.io/
export API_V2_ENABLED=true
```

These are the logs 
`2023-12-11T13:33:40.790 application=que [info] [Que] Queued new Job for Explorer.SmartContract.Solidity.PublisherWorker
2023-12-11T13:33:40.791 application=que [info] [Que] Starting Job # 42 with Explorer.SmartContract.Solidity.PublisherWorker
2023-12-11T13:33:40.791 application=api request_id=F5_KTg-IKjBPfQoAAZkF [info] Sent 204 in 663µs
2023-12-11T13:33:40.803 application=que [info] [Que] Completed Job # 42 with Explorer.SmartContract.Solidity.PublisherWorker
`

**Comment by ekawibisono** [comment]
Hello @L2nitant You Build blockscout using Docker-compose or manualy ?

**Reply by L2nitant** [reply]
blockscout manually. I used blockscout-rs and for smart contract verifier, docker is used 
Command is  
`docker run -p 8050:8050 ghcr.io/blockscout/smart-contract-verifier:latest`

Also Please confirm that is it important to run smart contract verifier before starting the blockscout or we can also do it after blockscout has started working?

**Reply by ekawibisono** [reply]
alright, you run using methode manualy right, what your OS run Blockscout ? @L2nitant

**Reply by L2nitant** [reply]
it is on aws server

**Reply by ekawibisono** [reply]
Yes AWS OS ?

**Reply by L2nitant** [reply]
Linux
